stdlib/3/concurrent/futures/process.pyi

```diff
@@ -7,5 +7,11 @@ EXTRA_QUEUED_CALLS = ...  # type: Any
 if sys.version_info >= (3,):
     class BrokenProcessPool(RuntimeError): ...
 
-class ProcessPoolExecutor(Executor):
-    def __init__(self, max_workers: Optional[int] = ...) -> None: ...
+if sys.version_info >= (3, 7):
+    class ProcessPoolExecutor(Executor):
+        def __init__(self, max_workers: Optional[int] = ...,
+                     initializer: Callable[..., None] = ...,
+                     initargs: Tuple[Any] = ...) -> None: ...
```

Comment:
Should be `Tuple[Any, ...]` (a tuple of any number of elements of any type; the current code means a tuple of one element).